client/src/pages/MainMenu.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styled from "@emotion/styled";
+import Header from "../components/HeaderGoBack";
+import MenuAvailableSrc from "../assets/menu-available.svg";
+import MenuNotAvailableSrc from "../assets/menu-not-available.svg";
+import MenuRentedSrc from "../assets/menu-rented.svg";
+import MenuDueDateSrc from "../assets/menu-due-date.svg";
+import MenuNewArticleSrc from "../assets/menu-new-article.svg";
+import MenuNewRentSrc from "../assets/menu-new-rent.svg";
+
+function MainMenu() {
+  return (
+    <>
+      <Header title={"Menü"} />
+      <Container>
+        <ul>
+          <li>
+            <img src={MenuAvailableSrc} alt="menu-avaiblable" />
+            Artikel auf Lager
+          </li>
```

Comment:
It might be possible to create a List and Listitem component to reduce the lines of code in this file.